test/tools_test.cc

```diff
@@ -47,7 +49,11 @@ constexpr double kLaneWidth{3.7};
 constexpr double kShoulderWidth{3.};
 constexpr double kMaximumHeight{5.2};
 
+constexpr char MALIPUT_MALIDRIVE_RESOURCE_ROOT[] = "MALIPUT_MALIDRIVE_RESOURCE_ROOT";
+constexpr char MULTILANE_RESOURCE_ROOT[] = "MULTILANE_RESOURCE_ROOT";
+
 GTEST_TEST(CreateRoadNetwork, MalidriveRoadNetwork) {
+  setenv(MALIPUT_MALIDRIVE_RESOURCE_ROOT, DEF_MALIDRIVE_RESOURCES, 1);
```

Comment:
Let's set and unset these variables in a set up and tear down of the test. Also, let's replace the 1 by a named constant "kOverwrite" so it is clear that we want a non-zero value for that purpose and it is more readable.

I am asking for the set up and tear down so we make sure that those variables are set and unset for the required sets and there is no other environment leakage that blows in the future. These things are usually hard to solve.

Reply:
Makes sense. 
Done.